apps/meteor/client/views/admin/users/AdminUserInfoWithData.tsx

```diff
@@ -107,7 +108,7 @@ const AdminUserInfoWithData = ({ uid, onReload }: AdminUserInfoWithDataProps): R
 					isAdmin={data?.user.roles.includes('admin')}
 					userId={data?.user._id}
 					username={user.username}
-					isAFederatedUser={data?.user.federated}
+					isAFederatedUser={isUserFederated(data?.user as unknown as IUser)}
```

Comment:
Try to be consistent in naming properties and methods and don't use articles like "a and an", instead be direct like `isFederatedUser`

Reply:
And why it was necessary to use `as unknown` since the method expects IUser as a param?

Reply:
Because the `data.user` is not a `IUser` type, there are some properties incompatible with `IUser` (e.g `createdAt` as string instead of Date)